media/css/seahub.css

```diff
@@ -93,6 +93,58 @@
 }
 .sf-icon-share:before { content:"\e001"; }
 .sf-icon-removeformat:before { content:"\f1e7"; }
+/****** sf2-icon-xx ********/
+@font-face {
+    font-family: 'seafile-font2';
+    src:url('sf_font2/myFont.eot');
+    src:url('sf_font2/myFont.eot?#iefix') format('embedded-opentype'),
+        url('sf_font2/myFont.woff') format('woff'),
+        url('sf_font2/myFont.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+[class^="sf2-icon-"] {
+    font-family: 'seafile-font2';
+    speak: none;
+    font-weight: normal;
+    font-variant: normal;
+    text-transform: none;
+    -webkit-font-smoothing: antialiased;
+}
+.sf2-icon-bell:before { content:"\e600"; }
+.sf2-icon-share:before { content:"\e601"; }
+.sf2-icon-download:before { content:"\e602"; }
+.sf2-icon-snow:before { content:"\e603"; }
+.sf2-icon-delete:before { content:"\e604"; }
+.sf2-icon-circle:before { content:"\e605"; }
+.sf2-icon-delete-o:before { content:"\e606"; }
```

Comment:
2 delete? sf2-icon-delete, sf2-icon-delete-o.

and what do you mean by sf2-icon-delete-o?